### PR TITLE
fix(@angular-ru/ng-table-builder): fix null values filter errors

### PR DESCRIPTION
**Pull request by Schirbak**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
-   [x] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: packages/ng-table-builder/src/services/filterable/filter.worker.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ type NumericFilterTypes =
     | TableFilterType.LESS_THAN
     | TableFilterType.MORE_THAN;
 
-type PlainValue = string | number | boolean;
+type PlainValue = string | number | boolean | null | undefined;
```
Comment (splincode) [inline packages/ng-table-builder/src/services/filterable/filter.worker.ts:13]: `type PlainValue = Nullable<string | number | boolean>`

**Review by splincode — CHANGES_REQUESTED** [review]

File: packages/ng-table-builder/src/services/filterable/filter.worker.ts [hunk 2]
```diff
@@ -78,7 +78,7 @@ export function filterAllWorker<T>({ source, global, types, columns }: Filterabl
     }
 
     function toLowercase(value: PlainValue): string {
-        return value.toString().trim().toLocaleLowerCase();
+        return (value ?? '').toString().trim().toLocaleLowerCase();
```
Comment (splincode) [inline packages/ng-table-builder/src/services/filterable/filter.worker.ts:81]: `value?.toString().trim().toLocaleLowerCase() ?? ''`;

Comment (Schirbak) [reply packages/ng-table-builder/src/services/filterable/filter.worker.ts]: done

File: packages/ng-table-builder/src/services/filterable/filter.worker.ts (hunk 1 above)
Comment (Schirbak) [reply packages/ng-table-builder/src/services/filterable/filter.worker.ts]: done

**Review by mansurishmuhametov — APPROVED** [review]

**Review by splincode — APPROVED** [review]

**Pull request merged**